Makefile

```diff
@@ -1,6 +1,6 @@
 export
 
-# Make sure that sort always uses the same sort order.
+## Make sure that sort always uses the same sort order.
 LC_ALL := C
```

Comment:
I am curious: what do the double `##` signify?

Reply:
[`makefile-parser`](https://github.com/kba/makefile-parser) won't pick it up as a comment on that variable and it won't be listed in the [help target](https://github.com/kba/makefile-parser#make-help).